tests/testthat/test-run-log.R

```diff
@@ -2,9 +2,8 @@ context("Constructing run log from model yaml")
 
 source("data/test-workflow-ref.R")
```

Comment:
We should change from the `source()` pattern to the `testthat` handling for special files, as described in metrumresearchgroup/mrgvalidatetestreference#8 (at least for all new development, if we don't want to change everything right now).